README.md

```diff
@@ -1,15 +1,19 @@
-# Smartsheet Java SDK
+# Smartsheet SDK for Java
 
 This is a Java SDK to simplify connecting to [Smartsheet API](http://www.smartsheet.com/developers/api-documentation) in Java applications.
 
+## System Requirements
+
+The SDK supports Java version 1.6 or later.
+
 ## Installation
 There are three different ways to install the SDK. Select the one that fits your environment best:
 
-1. [Use Maven](#1-maven)
-2. [Download Jar File](#2-download-jar-file)
-3. [Compile From Source](#3-compile-from-source)
+* [Install By Using Maven](install-by-using-maven)
```

Comment:
These hyperlinks are broken in the preview. Please make sure they work when published.